mdsuite/utils/calculator_helper_methods.py

```diff
@@ -160,7 +163,7 @@ def func(x, m, a):
 #                 > self.experiment.number_of_configurations
 #                 - self.args.correlation_time
 #             ):
-#                 print("Trajectory not long enough to perform analysis.")
+#                 log.errorgit stat("Trajectory not long enough to perform analysis.")
```

Comment:
something went wrong here - not important because commented out

Reply:
good catch. looks like my cursor was in the wrong window :)